src/central-submissions.rst

```diff
@@ -143,3 +147,52 @@ To connect with Excel or Power BI, follow these steps.
++------------+-------------+-----------------------------------------------------------------------------------+
+| Approved   | User        | Can be assigned by project staff to approve a submission                          |
++------------+-------------+-----------------------------------------------------------------------------------+
+| Rejected   | User        | Can be assigned by project staff to reject a submission                           |
++------------+-------------+-----------------------------------------------------------------------------------+
+
+The ``Received`` and ``Edited`` states are automatically set by Central any time a submission is uploaded or edited. The other states are assigned by project staff. Review states have suggested meanings, but *they do not do anything by themselves*. You are free to use these states however you'd like.
+
+Once submissions have been reviewed, the submission table download and the OData connection both allow submissions to be filtered by review state. This lets you, for example, download only all the approved submissions.
+
+.. _central-submissions-details:
+
+Submission Details
+------------------
+
+As of version 1.2, each submission has its own detail page which provides basic information about the submission, an activity history of action and discussion on that submission, and tools for updating the submission review state and data itself.
+
+   .. image:: /img/central-submissions/details.png
+
+The title at the top is pulled from the ``instanceName`` metadata tag if there is one, otherwise it will be the ``instanceID``. You can set up your form to compute an ``instanceName`` based on the data in each submission. You may want to do this if you plan on using this page a lot, because it makes navigation much easier to have friendly names at the top of the page and in the web browser title and tab.
+
+Basic detail can be found along the left. If there are expected media attachments for this submission, that status information will be provided.
+
+The main activity feed on the right shows you the discussion and action history of the submission. Any review state changes, comments, and edits will appear here. At the top of the activity feed, you can :guilabel:`Review` a submission to assign a new review state, :guilabel:`Edit` the submission directly using Enketo in your web browser, or type in the box to begin leaving a comment.
+
+You can leave a note when you update the review state, to indicate why the decision is being made, or any other information you'd like saved.
+
+Any time a user makes a submission edit, they will see a loud note when they are returned to this detail page suggesting that they leave a comment describing the edits they have made. This is optional but highly encouraged. In a future version of Central, greater detail will be automatically provided about the data values that were changed.
```

Comment:
s/makes a submission edit/edits a submision/

s/loud//.

Reply:
(i put loud in there bc i know it's been a concern for people that this step is hard to miss)